include/proj/coordinateoperation.hpp

```diff
@@ -148,6 +148,8 @@ class PROJ_GCC_DLL CoordinateOperation : public common::ObjectUsage,
     PROJ_DLL bool
     isPROJInstanciable(const io::DatabaseContextPtr &databaseContext) const;
 
+    PROJ_DLL bool hasRoughTransformation() const;
```

Comment:
I'm probably not a good enough English speaker to be a judge of this but "rough" sounds a bit off to me. I understand the intention it just doesn't seem like the right word to me for some reason. Maybe "approximate" or "imprecise"?

Reply:
Also, it sounds a bit like "Hough transformation" and that's something entirely different :)

Reply:
how about `hasApproximateTransformation`?

Reply:
yeah, I wasn't sure about the appropriate term. All transformations are "approximate" / "imprecise" by nature. I also considered "crude". @mwtoews any suggestion of which term might convey the information that this is *very* approximate ?

Reply:
yeah, English doesn't have relative words for _more_ or _less_ approximate. All I can think of is a list of other words similar to rough, such as ballpark, fuzzy, etc.

Reply:
Hum, maybe hasUnpreciseTransformation() ?

Reply:
Ok, it's silly, but I'm appreciating the adjective ["ballpark"](https://www.collinsdictionary.com/dictionary/english/ballpark) at the moment. It's illustrative of spatial inaccuracies by it's very definition, which in some cases (e.g. ~40 m) is in the range of the size of a ballpark.

Reply:
@kbevers Are you fine with ballpark ?

Reply:
Yes, I think it is appropriate to include a bit of silliness from time to time :-)

Reply:
In network adjustments you would start out with "preliminary coordinates" before refining through the adjustment, so perhaps a "preliminary transformation" would work?

And while doing linguistic nitpicking,  I think "instanciation" and multiple other derived forms all over the code tree should be spelled "instantiation", i.e. with a "t". Probably most important to correct this in function\method names.